app/views/pages/_contributor_blurb.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="gridManyItem center">
+  <%= react_component('Modal', props: {
+    element: { 
+      component: 'Avatar', 
+      props: { 
+        src: ProfilePicture.normalize_url(blurb["image"]),
+        alt: t('pages.contribute.contributor_avatar_alt', {
+          name: blurb['name']
+        })
+      } 
+    },
+    body: <<~HEREDOC
+      <div class="contributorModal-body">
+        <div class="contributorModal-body-photo">
+          <div class="avatar Avatar__avatar___3vcX_ Avatar__large___1dps1">
+            <img class="Avatar__image___2cXeU" src="#{ProfilePicture.normalize_url(blurb["image"])}">
+          </div>
+        </div>
+
+        <div class="contributorModal-body-copy">
+          <h4 class="contributorModal-name">
+            #{blurb["name"]}
+          </h4>
+          <div class="contributorModal-body-socialLinks">
+            <i class="fa fa-location-arrow fa-inline"></i>#{blurb["location"]}
```

Comment:
Let's reduce the space to the right of this element so it matches the first icon in this section.